src/Symfony/Component/Translation/LocaleAwareInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation;
+
+/**
+ * LocaleAwareInterface should be implemented by classes that needs the current locale
+ *
+ * @author Pierre Bobiet <pierrebobiet@gmail.com>
+ */
+interface LocaleAwareInterface
```

Comment:
im not sure this belongs here, what about a new contract `Contract\Service\LocalizedInterface`?

Reply:
I'll do that :)